x-pack/plugins/maps/public/connected_components/layer_addpanel/view.js

```diff
@@ -95,87 +85,92 @@ export class AddLayerPanel extends Component {
   };
 
   _layerAddHandler = () => {
-    const {
-      isIndexingTriggered,
-      setIndexingTriggered,
-      selectLayerAndAdd,
-      resetIndexing,
-    } = this.props;
```

Comment:
thx for removing destructuring. code is so much more readable when context is preserved at call-site.